tests/test_ci/TestControllers.py

```diff
@@ -678,7 +681,7 @@ def test_webhook_push_no_after(self, mock_request):
     @mock.patch('mod_ci.controllers.GeneralData')
     def test_webhook_push_valid(self, mock_gd, mock_github, mock_add_test_entry, mock_request):
         """Test webhook triggered with push event with valid data."""
-        data = {'after': 'abcdefgh'}
+        data = {'after': 'abcdefgh', 'ref': 'refs/heads/master'}
```

Comment:
If the string - `'refs/heads/master'` - is used multiple times, let's move it to a constant string and use it.

Reply:
Just once in this file...